src/kvstore/raftex/RaftPart.cpp

```diff
@@ -1211,7 +1211,7 @@ void RaftPart::statusPolling(int64_t startTime) {
         } else {
             // No leader has been elected, sleep for a while to continue
             VLOG(2) << idStr_ << "Wait for a while and continue the leader election";
-            delay = FLAGS_raft_heartbeat_interval_secs * 1000;
+            delay = FLAGS_raft_heartbeat_interval_secs * 1000 + folly::Random::rand32(500);
```

Comment:
Does this avoid the problem of all nodes voting at the same time?

Reply:
Yeah, I delete it in #2297 by mistake.